src/pages/en/getting-started-new.md

```diff
@@ -0,0 +1,112 @@
+---
+setup: |
+    import Button from '../../components/Button.astro'
+layout: ~/layouts/MainLayout.astro
+title: Getting Started
+description: A basic intro to Astro.
+---
+Static Site Generator  🚀  Bring your own Framework  🚀  Ship Less JavaScript
+
+## Try Astro
+
+We've made it as easy as possible to get started with Astro either in your browser, or on your machine!
+
+### Online Playgrounds
+
+Visit [astro.new](https://astro.new) for the easiest way to "try before you buy." Choose from a variety of starter templates and start building a full, working version of Astro right in your browser!
+
+Or, **instantly launch our basic starter project** with just one click of a button:
+
+<!-- TODO: Include a link in the first few words of this paragraph to `astro.new`, so that anyone skimming it sees it.
```

Comment:
left these TODOs for you to take a look at for copy!

Reply:
I *think* these todo's were already addressed.... (I might have moved them further away from the things they actually refer to... But every TODO from last night I think I got!)